terraform/github/modules/labels/README.md

```
@@ -34,6 +36,20 @@ module "github_labels" {
 }
 ```
 
+### Automated Usage
+
+This module can be automatically applied to repositories using the [GitHub Labels workflow](../../../../.github/workflows/github-labels.yml).
+
+To run the workflow:
+1. Go to the Actions tab in the repository
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)